src/components/topic/Description.js

```diff
@@ -1,22 +1,49 @@
 import React from 'react'
+import YoutubePlayer from 'react-youtube'
+import constStyledComponents from '../../constants/styled-components'
+import getArticleComponent from '../article/getArticleComponent'
 import styled from 'styled-components'
 import PropTypes from 'prop-types'
 import { lineHeight, typography, colors } from '../../themes/common-variables'
+// lodash
+import get from 'lodash/get'
 
+const fontSize = 'medium'
 const redLineDistance = '40px'
+const contentSize = 'small'
+
+const _ = {
+  get
+}
+
+const StyledTopicComponent = constStyledComponents.ResponsiveContainerForAritclePage.extend`
```

Comment:
Will remove style in `constStyledComponents.ResponsiveContainerForAritclePage` from `StyledTopicComponent` since layout width of topic landing page is inconsistent with our article page in tablet. I had not noticed that, it's my fault.